frontend/src/assets/variables.scss

```diff
@@ -1,19 +1,20 @@
 /* --------- variable overrides for primevue--------- */
 // ref: https://primevue.org/colors/
 :root {
-  --text-color: #495057;
-  --primary-color: #036;
+  // --text-color: #495057;
+  // --primary-color: #036;
 }
 
 /* -------- custom css variables --------- */
 // buttons, links
 $app-primary: #036;
+$app-error: #D8292F;
+
 $app-link-text: #1a5a96;
 $app-link-text-hover: #00f;
 $app-hover: #4696EC;
 $app-outline-on-primary: #fff;
 $app-out-of-focus: #ccc;
-$app-error: #D8292F;
 $app-green: #2e8540;
 $app-grey: #e9ecef;
```

Comment:
I believe some of these can be removed as they can either be accessible via `var(--p-primary-color)`, are just unused, or maybe exist once and can probably be replaced via the first method. Will leave it to you to evaluate.

Reply:
Moved all variables in this file to `ThemeOverride.ts` file. Fixed all references to new system. Removed extraneous `variables.scss` file.